ppanggolin/2.1.2/Dockerfile

```diff
@@ -0,0 +1,53 @@
+ARG PPANGGOLIN_VER="2.1.2"
+
+FROM mambaorg/micromamba:1.5.8 as app
+
+ARG PPANGGOLIN_VER
+
+# 'LABEL' instructions tag the image with metadata that might be important to the user
+LABEL base.image="mambaorg/micromamba:1.5.8"
+LABEL dockerfile.version="1"
+LABEL software="PPanGGOLiN"
+LABEL software.version="${PPANGGOLIN_VER}"
+LABEL description="Depicting microbial species diversity via a Partitioned PanGenome Graph Of Linked Neighbors"
+LABEL website="https://github.com/labgem/PPanGGOLiN"
+LABEL license="https://github.com/labgem/PPanGGOLiN/blob/master/LICENSE.txt"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+USER root
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    ca-certificates \
+    procps \
+    wget && \
+    apt-get autoclean && rm -rf /var/lib/apt/lists/*
+
+ENV PATH="/opt/conda/envs/base/bin/:/opt/conda/bin/:$PATH" LC_ALL=C
+
+RUN micromamba install --name base -c conda-forge -c bioconda -c defaults ppanggolin=${PPANGGOLIN_VER} && \
+    micromamba clean -a -y && \
+    ppanggolin -h && \
+    mkdir /data
+
```

Comment:
Please force package cache cleaning. 
I think lines 30 and 31 are also redundant here.

Reply:
I added the -f flag and removed lines 30 & 31

Thank you for the suggestions!